script/plan_viz2.py

```diff
@@ -15,27 +15,15 @@
 from plan_config2 import PlanConfig2
 from util import TASK_COLORS, AGENT_COLORS, DIR_OFFSET, TEXT_SIZE, \
     get_angle, get_dir_loc, get_rotation
-
+import platform
```

Comment:
import math
import platform
import time
import tkinter as tk
from tkinter import ttk
from typing import Dict, List, Set, Tuple

from plan_config2 import PlanConfig2
from util import (AGENT_COLORS, DIR_OFFSET, TASK_COLORS, TEXT_SIZE, get_angle,
                  get_dir_loc, get_rotation)

Reply:
I use an extension in VSCode called "Pylint" to check if there are any bad formats in my Python code.
Please also use this as well.